hw/ip/spi_device/data/spi_device.hjson

```diff
@@ -4,6 +4,12 @@
 {
   name:               "spi_device",
   one_line_desc:      "Serial peripheral interface supporting different device modes, suitable for bulk-load of data into and out of the chip",
+  one_paragraph_desc: '''
+  The SPI Device is a configurable, versatile hardware block that implements a generic SPI device mode, plus three additional modes (SPI Flash emulation mode, SPI passthrough mode, and TPM over SPI mode) to support a variety of different applications.
+  For example, generic mode can be used in a raw data transfer protocol termed "Firmware Operation Mode", which is intended to be used to bulk-load data into and out of the chip.
```

Comment:
bulk-load of data

Reply:
I think it's better without the 'of'? With the of you're saying a 'bulk-load' of data, i.e. there is data and bulk-load describes it. Without the 'of' bulk-load is the verb, you're 'bulk-loading' the data. The second meaning seems more correct.

Reply:
I agree with @GregAC. We wrote "for bulk-load *of* data" in the one-line description, but there we use 'bulk-load' as noun

Reply:
Makes sense, thanks for your explanations both.